Hudl.Mjolnir/Breaker/FailurePercentageCircuitBreakerConfig.cs

```diff
@@ -1,51 +1,60 @@
-﻿using Hudl.Mjolnir.External;
-using Hudl.Mjolnir.Key;
+﻿using Hudl.Mjolnir.Key;
 using System;
+using Hudl.Mjolnir.Config;
 
 namespace Hudl.Mjolnir.Breaker
 {
     internal class FailurePercentageCircuitBreakerConfig : IFailurePercentageCircuitBreakerConfig
     {
-        private readonly IMjolnirConfig _config;
+        private readonly MjolnirConfiguration _config;
 
-        public FailurePercentageCircuitBreakerConfig(IMjolnirConfig config)
+        public FailurePercentageCircuitBreakerConfig(MjolnirConfiguration config)
         {
             _config = config ?? throw new ArgumentNullException(nameof(config));
         }
 
+        private BreakerConfiguration _getBreakerConfiguration(GroupKey key)
```

Comment:
Method name should be `GetBreakerConfiguration` instead of `_getBreakerConfiguration`.